packages/oas/src/converter/subconverters/PostDataConverter.ts

```diff
@@ -0,0 +1,231 @@
+import { Sampler } from '../Sampler';
+import { isOASV2 } from '../../utils';
+import { SubConverter } from './SubConverter';
+import { OpenAPI, PostData } from '@har-sdk/core';
+import pointer from 'json-pointer';
+import { toXML } from 'jstoxml';
+import { stringify } from 'qs';
+
+export class PostDataConverter implements SubConverter<PostData | null> {
+  private readonly JPG_IMAGE = '/9j/2w==';
+  private readonly PNG_IMAGE = 'iVBORw0KGgo=';
+  private readonly BOUNDARY = '956888039105887155673143';
+  private readonly BASE64_PATTERN =
+    /^([A-Za-z0-9+/]{4})*([A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==)?$/;
+
+  constructor(
+    private readonly spec: OpenAPI.Document,
+    private readonly sampler: Sampler
+  ) {}
+
+  // eslint-disable-next-line complexity
+  public convert(path: string, method: string): PostData | null {
+    const pathObj = this.spec.paths[path][method];
+    const tokens = ['paths', path, method];
+    const params = Array.isArray(pathObj.parameters) ? pathObj.parameters : [];
```

Comment:
Why don't reuse `getParameters` and `filterLocationParams` functions to avoid if statement under for..of loop?

```suggestion
    const allParams: ParameterObject[] = getParameters(this.spec, path, method);
    const params = filterLocationParams(allParams, 'body');
    const tokens = ['paths', path, method];
```

Reply:
just because `PostDataConverter` is copied "as is" for now (and excluded from codeclimate)